components/Header/NavSubHeader/SortingOptions.vue

```diff
@@ -90,47 +82,61 @@ export default {
       else {
         return '';
       }
+    },
+    sort() {
+      let pageType;
+      switch(this.type) {
+        case '/p/':
+          pageType = 'project';
+          break;
+        case '/u/':
+          pageType = 'profile';
+          break;
+        case '/c/':
+          pageType = 'community';
+          break;
+        case '/':
+          pageType = 'newsfeed';
+          break;
+      }
+      return this.$store.state.sorting[pageType];
     }
   },
   created() {
-    // sort the current route and see what type of we're on
-    try { // look for non-newsfeed path
+    // sort the current route and see what type of
+    // page we're on
+    try {
       let re = new RegExp('/[u,p,c]/');
       this.type = this.$route.fullPath.match(re)[0];
```

Comment:
we should try to make this a utils function, because we use it in A LOT of components